### PR TITLE
DLUHC-175 Add radio buttons

### PR DESCRIPTION
**Pull request by bethpritchard**
Adds radio button input 

image

### CODE REVIEW

**Review by samornsby — APPROVED** [review]
LGTM

File: dluhc-component-library/src/components/siteSelectionForm/components/RadioButtons.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  }
+
+  const optionComponents = options.map((key) => (
+    <div key={key} className="flex items-center mb-4">
+      <label className="font-semibold flex">
+        <input
+          type="radio"
+          class="radio mr-2"
+          name={name}
+          checked={value == key}
```
[the remark this answers is not in the document]
Comment (bethpritchard) [reply dluhc-component-library/src/components/siteSelectionForm/components/RadioButtons.tsx]: Done

File: dluhc-component-library/src/components/siteSelectionForm/components/DynamicForm.tsx [hunk 2]
```diff
@@ -14,6 +15,7 @@ interface DynamicFormProps {
 enum InputType {
   TextInput,
   NumberInput,
+  RadioInput,
```
[the remark this answers is not in the document]
Comment (bethpritchard) [reply dluhc-component-library/src/components/siteSelectionForm/components/DynamicForm.tsx:18]: I think we only have one at the moment so gonna leave it

File: dluhc-component-library/src/components/siteSelectionForm/components/DynamicForm.tsx [hunk 3]
```diff
@@ -25,6 +27,9 @@ const getInputType = (formPageSchema: FormPageSchema) => {
   if (formPageSchema.type === "number") {
     return InputType.NumberInput;
   }
+  if (formPageSchema.type === "radio") {
```
Comment (amjones-scottlogic) [inline dluhc-component-library/src/components/siteSelectionForm/components/DynamicForm.tsx:30]: Worth noting "radio" isnt a valid jsonSchema type but until we add uiSchema this is a work around

**Review by amjones-scottlogic — APPROVED** [review]

**Pull request merged**